node/lib/packaging/PackageInfo.ts

```diff
@@ -0,0 +1,173 @@
+
+/**
+ * Information about an OpenT2T package, as loaded from the package.json.
+ */
+export class PackageInfo {
```

Comment:
As we are allowing any 3rd party to publish, shouldn't we have a publisher name (& publisher display name) ?